bundles/framework/divmanazer/component/SelectList.js

```diff
@@ -66,6 +74,21 @@ Oskari.clazz.define('Oskari.userinterface.component.SelectList',
       });
       return el;
     },
+    /**@method makeMultiChosen
+    *  applies jQuery chosen to specidied element
+    * @param {element} el
+     */
+    makeMultiChosen: function( el, options ) {
+      el.find( 'select' ).attr('multiple', true);
```

Comment:
Why not just move the options.multi logic to makeChosen() function? This is basically a copy pasted version of the method with one line difference and 1 to 5 lines on the caller side :)

Reply:
Remove this function, make all the callers use makeChosen() as they did before and add this at the beginning of makeChosen() :

    if(options.multi) {
        el.find( 'select' ).attr('multiple', true);
    }